engineering/workflow.md

```diff
@@ -20,10 +18,11 @@ Date: 2018-01-15
 
 ## Intro
 
-These are the notes from:
+These based in the notes from:
```

Comment:
These what? Should it be 'This document is based on the notes from:'?

Reply:
Good catch, thanks.